### PR TITLE
ENH support master or main for tooling

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Added a ``news`` entry

This PR adds support for master or main as the default branch for the tooling repos.

### CODE REVIEW

File: news/main_feedstock.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+**Added:**
+
+* Enabled fully changing the default branch from ``master`` to ``main``.
```
Comment (beckermr) [inline news/main_feedstock.rst:3]:
```suggestion
```

**Comment by beckermr** [comment]
@conda-forge/core this one is ready for review!

**Comment by beckermr** [comment]
Bump here @conda-forge/core

**Review by chrisburr — APPROVED** [review]
Sorry I thought I'd already approved this one :+1:

**Comment by beckermr** [comment]
Thanks @chrisburr!

**Pull request merged**